emails/tracker_lists/general-tracker.json

```diff
@@ -0,0 +1,216 @@
+[
+  "10web.io",
```

Comment:
Oh, so these lists are coming from shavar-prod-lists' blocklist [here](https://github.com/mozilla-services/shavar-prod-lists/blob/master/disconnect-blacklist.json). So ideally we may want to do a github action to either (1) parse the list to create files on our repo (2) submodule the entire repo and parse when service starts. We may end up having to do (2) because of license reasons but while we figure out long term solution and not block testing I decided to add it to our repo.

Not sure about how often the list gets uploaded but for testing we should use a static list (test/study is expected to end on May 4th and this is when I hope to have a long term solution for maintaining this list.